### PR TITLE
CORE-1199: generate avax templates

### PR DESCRIPTION
**Pull request by AmitBRD**
Add generated avax files

### CODE REVIEW

**Review by EBGToo — DISMISSED** [review]

File: WalletKitJava/WalletKit/src/main/java/com/blockset/walletkit/NetworkType.java [hunk 1]
```diff
@@ -18,5 +18,5 @@
     HBAR,
     XTZ,
     XLM,
-    /* __SYMBOL__ */
+    AVAX,n    /* __SYMBOL__ */
```
Comment (bgoring) [inline WalletKitJava/WalletKit/src/main/java/com/blockset/walletkit/NetworkType.java:21]: this and below still have the newline issue, wont compile...

**Review by bgoring — APPROVED** [review]

File: WalletKitJava/WalletKit/src/main/java/com/blockset/walletkit/NetworkType.java (hunk 1 above)
Comment (EBGToo) [reply WalletKitJava/WalletKit/src/main/java/com/blockset/walletkit/NetworkType.java:21]: @AmitBRD please have a look and update!

File: WalletKitCore/WalletKitCoreTests/WalletKitCoreTests.swift [hunk 2]
```diff
@@ -464,6 +467,6 @@ final class WalletKitCoreTests: XCTestCase {
         ("testSyncOneBTC",      testBitcoinSyncOne),
 //        ("testManaagerSyncBTC", testBitcoinWalletManagerSync)
         
-        // __NEW_BLOCKCHAIN_TEST__
+        // Avalanchen        ("testAvalanche", testAvalanche),n        // __NEW_BLOCKCHAIN_TEST__
```
Comment (EBGToo) [inline WalletKitCore/WalletKitCoreTests/WalletKitCoreTests.swift:470]: Still a problem?

File: WalletKitCore/include/WKBase.h [hunk 3]
```diff
@@ -276,10 +276,10 @@ typedef enum {
     WK_NETWORK_TYPE_HBAR,
     WK_NETWORK_TYPE_XTZ,
     WK_NETWORK_TYPE_XLM,
-    /* WK_NETWORK_TYPE___SYMBOL__ */
+    WK_NETWORK_TYPE_AVAX,n    /* WK_NETWORK_TYPE___SYMBOL__ */
```
Comment (EBGToo) [inline WalletKitCore/include/WKBase.h:279]: Still a problem?

File: WalletKitCore/include/WKBase.h [hunk 4]
```diff
@@ -298,7 +298,7 @@ typedef enum {
 #define WK_NETWORK_CURRENCY_HBAR    "hbar"
 #define WK_NETWORK_CURRENCY_XTZ     "xtz"
 #define WK_NETWORK_CURRENCY_XLM     "xlm"
-/* #define WK_NETWORK_CURRENCY___SYMBOL__    "__symbol__" */
+#define WK_NETWORK_CURRENCY_AVAX    "avax" n/* #define WK_NETWORK_CURRENCY___SYMBOL__    "__symbol__" */
```
Comment (EBGToo) [inline WalletKitCore/include/WKBase.h:301]: Still here?

File: WalletKitCore/src/walletkit/WKConfig.h [hunk 5]
```diff
@@ -300,7 +300,7 @@ DEFINE_MODES            ("stellar-testnet", WK_SYNC_MODE_API_ONLY)
...
...avalanche-mainnet",     "avalanche-mainnet:__native__",   NETWORK_NAME,  WK_NETWORK_CURRENCY_AVAX,  "native",   NULL,   true)n    DEFINE_UNIT ("avalanche-mainnet:__native__",  "lumen_i",     "xlm_i",   0,  "AVAX_I")n    DEFINE_UNIT ("avalanche-mainnet:__native__",  "lumen",       "xlm",     7,  "AVAX")nDEFINE_ADDRESS_SCHEMES  ("avalanche-mainnet", WK_ADDRESS_SCHEME_NATIVE)nDEFINE_MODES            ("avalanche-mainnet", WK_SYNC_MODE_API_ONLY)nn#if HAS_AVAX_TESTNETnDEFINE_NETWORK (WK_NETWORK_TYPE_AVAX,  "avalanche-testnet", NETWORK_NAME, "testnet", false, 1075721, 1, 5)nDEFINE_NETWORK_FEE_ESTIMATE ("avalanche-testnet", "100", "5s", 5 * 1000)nDEFINE_CURRENCY ("avalanche-testnet",     "avalanche-testnet:__native__",   NETWORK_NAME,  WK_NETWORK_CURRENCY_AVAX,  "native",   NULL,   true)n    DEFINE_UNIT ("avalanche-testnet:__native__",  "lumen_i",     "txlm_i",   0,  "tAVAX_I")n    DEFINE_UNIT ("avalanche-testnet:__native__",  "lumen",       "txlm",     7,  "tAVAX")nDEFINE_ADDRESS_SCHEMES  ("avalanche-testnet", WK_ADDRESS_SCHEME_NATIVE)nDEFINE_MODES            ("avalanche-testnet", WK_SYNC_MODE_API_ONLY)n#endifnDEFINE_HANDLERS (WK_NETWORK_TYPE_AVAX, AVAX)n#undef NETWORK_NAMEnn// __CONFIG__
```
Comment (EBGToo) [inline WalletKitCore/src/walletkit/WKConfig.h:303]: This too?  @bgoring what is happening here?

File: WalletKitJava/WalletKitBRD/src/main/java/com/blockset/walletkit/brd/Utilities.java [hunk 6]
```diff
@@ -215,7 +215,7 @@ static WKNetworkType networkTypeToCrypto(NetworkType type) {
             case HBAR:return WKNetworkType.HBAR;
             case XTZ: return WKNetworkType.XTZ;
             case XLM: return WKNetworkType.XLM;
-            /* case __SYMBOL__: return WKNetworkType.__SYMBOL__; */
+            case AVAX: return WKNetworkType.AVAX; n            /* case __SYMBOL__: return WKNetworkType.__SYMBOL__; */
```
Comment (EBGToo) [inline WalletKitJava/WalletKitBRD/src/main/java/com/blockset/walletkit/brd/Utilities.java:218]: Here?

**Review by EBGToo — CHANGES_REQUESTED** [review]
I'm confused; was something reverted?  I thought there were only two errors; but the errors are all over.  Is it compiling?

File: WalletKitCore/CMakeLists.txt [hunk 7]
```diff
@@ -501,9 +501,9 @@ if (CMAKE_BUILD_TYPE MATCHES Debug)
             ${PROJECT_SOURCE_DIR}/WalletKitCoreTests/test/stellar/testStellar.c)
 endif(CMAKE_BUILD_TYPE MATCHES Debug)
 
-# __NEW_BLOCKCHAIN_IMPL__
+# Avalanche ntarget_sources (WalletKitCoren        PRIVATEn        ${PROJECT_SOURCE_DIR}/src/avalanche/BRAvalanche.hn        ${PROJECT_SOURCE_DIR}/src/avalanche/BRAvalancheAccount.cn        ${PROJECT_SOURCE_DIR}/src/avalanche/BRAvalancheAccount.hn        ${PROJECT_SOURCE_DIR}/src/avalanche/BRAvalancheAddress.cn        ${PROJECT_SOURCE_DIR}/src/avalanche/BRAvalancheAddress.hn        ${PROJECT_SOURCE_DIR}/src/avalanche/BRAvalancheBase.hn        ${PROJECT_SOURCE_DIR}/src/avalanche/BRAvalancheFeeBasis.cn        ${PROJECT_SOURCE_DIR}/src/avalanche/BRAvalancheFeeBasis.hn        ${PROJECT_SOURCE_DIR}/src/avalanche/BRAvalancheTransaction.cn        ${PROJECT_SOURCE_DIR}/src/avalanche/BRAvalancheTransaction.hn)n        n# __NEW_BLOCKCHAIN_IMPL__
 
-# __NEW_BLOCKCHAIN_TEST__
+# Avalanche Testsnif (CMAKE_BUILD_TYPE MATCHES Debug)n    target_sources (WalletKitCoreTestn            PRIVATEn            ${PROJECT_SOURCE_DIR}/WalletKitCoreTests/test/avax/testAvalanche.cn)nendif(CMAKE_BUILD_TYPE MATCHES Debug)n        n# __NEW_BLOCKCHAIN_TEST__
```
Comment (EBGToo) [inline WalletKitCore/CMakeLists.txt:506]: Aren't all of these wrong?  @bgoring ?

Comment (bgoring) [reply WalletKitCore/CMakeLists.txt:506]: Yes it looks again that the newline is being processed incorrectly.

**Review by EBGToo — APPROVED** [review]

**Pull request merged**